content/accessibility/descriptive-buttons.mdx

```diff
@@ -51,7 +51,7 @@ When reviewing buttons on a page, ensure the following are true:
   </Do>
   <Dont>
     <Code className="language-html">
-      {`<button type="button">OK</button>`}
+      {`<button aria-label="button" type="button">OK</button>`}
```

Comment:
Thank you for contributing to these docs! ✨

One suggested change: Since the caption says “redundant”, let’s make the `aria-label` text match the button’s existing label:
```suggestion
      {`<button aria-label="OK" type="button">OK</button>`}
```

Reply:
Pleasure is all mine!
Yes, it makes more sense to use that instead.